server/clickhouse/schema.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE IF NOT EXISTS tool_logs
+(
+    ts              DateTime64(3, 'UTC'),
+    organization_id LowCardinality(String),
+    project_id      LowCardinality(String),
+    deployment_id   LowCardinality(String),
+    tool_type       Enum8('http'=1,'function'=2),
```

Comment:
```suggestion
    tool_type       LowCardinality(String),
```
Keep it open ended for now. Changing enums requires db migrations.